data/campaigns/Liberty/achievements.cfg

```diff
@@ -1 +1,40 @@
 #textdomain wesnoth-l
+[achievement_group]
+    display_name=_"Liberty"
+    content_for=liberty
+    [achievement]
+        id="liberty_or_death"
+        name=_"Give Me Liberty or Give Me Death!"
+        description=_"Complete Civil Disobedience with at least 3 peasants remaining."
+        icon="data/core/images/attacks/pitchfork.png"
+        icon_completed="data/core/images/attacks/pitchfork.png~SCALE(72,72)~BLIT("data/core/images/misc/achievement-frames/frame-3-red.png",0,0)"
+    [/achievement]
+    [achievement]
+        id="liberty_nameless"
+        name=_"The Nameless General"
+        description=_"Find out the name of the general in Unlawful Orders."
+        icon="data/core/images/icons/helmet_chain-coif.png"
+        icon_completed="data/core/images/icons/helmet_chain-coif.png~SCALE(72,72)~BLIT("data/core/images/misc/achievement-frames/frame-8-grey.png",0,0)"
+    [/achievement]
+    [achievement]
+        id="liberty_iron"
+        name=_"Harper vs. Goliath"
+        description=_"Defeat an Iron Mauler with Harper in Hide and Seek."
+        icon="data/core/images/attacks/slingshot.png"
+        icon_completed="data/core/images/attacks/slingshot.png~SCALE(72,72)~BLIT("data/core/images/misc/achievement-frames/frame-3-red.png",0,0)"
+    [/achievement]
+    [achievement]
+        id="liberty_hunter"
+        name=_"Perfect Hunter"
```

Comment:
A `# po:` hint that this corresponds to the level name would be good, in the translation tool it'll be detached from the `description=` line.

Reply:
Uhm, they're just translating the word hunter? It's not a special word.

Reply:
They, and that might mean someone else on the same translation team, are also translating the scenario name "The Hunters", the `# po:` comment would be a hint to use similar translations for both.